PodTemplates.d/release-linux.yaml

```diff
@@ -7,15 +7,13 @@ metadata:
 spec:
   containers:
     - name: jnlp
-      image: jenkinsciinfra/packaging:2.1.9
+      image: jenkinsciinfra/packaging:2.1.41
```

Comment:
needs a backport of https://github.com/jenkinsci/jenkins/pull/6628 to 2.332 and 2.346

Reply:
Could this change be (partially) reverted instead?

Reply:
> > packaging branch will also need merging then
> 
> I'm not aware of any `packaging` branch, what are you refering to?

https://github.com/jenkinsci/packaging/branches

> Could this change be (partially) reverted instead?

Yes it would just need this version bump reverted

Reply:
https://github.com/jenkins-infra/release/pull/252

Reply:
Hop: https://github.com/jenkins-infra/release/pull/252